v4/test/graphql/graphql_test.go

```diff
@@ -16,6 +17,37 @@ import (
 )
 
 func TestGraphQL_integration(t *testing.T) {
+	t.Run("RawGraphQL", func(t *testing.T) {
+		t.Run("up", func(t *testing.T) {
+			err := testenv.SetupLocalWeaviate()
+			if err != nil {
+				t.Fatalf("failed to setup weaviate: %s", err)
+			}
+		})
+
+		var client *weaviate.Client
+		t.Run("createdb", func(t *testing.T) {
+			client = testsuit.CreateTestClient()
+			testsuit.CreateTestSchemaAndData(t, client)
+		})
+
+		t.Run("get all", func(t *testing.T) {
+			resultSet, gqlErr := client.GraphQL().Raw("{Get {Pizza {name}}}").Do(context.Background())
+			assert.Nil(t, gqlErr)
+
+			get := resultSet.Data["Get"].(map[string]interface{})
+			pizza := get["Pizza"].([]interface{})
+			assert.Equal(t, 4, len(pizza))
+		})
```

Comment:
You can actually just move this test inside `t.Run("Get")` there's already a DB being created and populated no need to do this once again in a separate test

Reply:
@donomii you want to leave this one small separately in `t.Run("RawGraphQL",`, yes?

Reply:
bc I suggested that it's so small that you can just move it inside the `t.Run("Get")` suite to be run there